sound/soc/sof/Kconfig

```diff
@@ -258,6 +258,10 @@ config SND_SOC_SOF_DEBUG_IPC_KERNEL_INJECTOR
 	  Say Y if you want to enable the IPC kernel injector.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_DEBUG_FW_GDB
+	tristate "SOF enable Firmware GDB debugging"
+	select SND_SOC_SOF_CLIENT
+
```

Comment:
In general I think this file needs some refinement around the tunnel protocol handling.

Reply:
@lgirdwood sorry, which tunnel protocol do you mean?

Reply:
I mean the copying/handling of GDB data to/from IPC layer, but I think you've already done this in the later patches in this PR.

Reply:
@lgirdwood gdb communication isn't done via IPCs, it's done via a memory window. Only the first "switch to gdb" command is an IPC, and that isn't a part of the actual gdb messaging yet.

Reply:
ok, but then how would stop/break work if there is no IRQ ?

Reply:
@lgirdwood that's what the added polling periodic delayed work does in https://github.com/thesofproject/linux/pull/5411/commits/551692fe8ac266ecd90efbeb462381c9f29f6142#diff-fdf6d136ee82a240596bbe084669793c8f1ca4499e85a85e92dba039fd81461aR310 . And it isn't only needed for breaks, it's needed for any replies to gdb commands